analyzers/its/expected/Nancy/S3900-Nancy-netstandard2.0.json

```diff
@@ -1260,12 +1260,6 @@
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy/Routing/Constraints/MinRouteSegmentConstraint.cs#L33",
       "Location": "Line 33 Position 35-45"
     },
-    {
-      "Id": "S3900",
-      "Message": "Refactor this method to add validation of parameter \u0027constraint\u0027 before using it.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy/Routing/Constraints/ParameterizedRouteSegmentConstraintBase.cs#L21",
```

Comment:
[Link](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy/Routing/Constraints/ParameterizedRouteSegmentConstraintBase.cs#L21)
As for case above, reason unclear.

Reply:
Discussed [above](https://github.com/SonarSource/sonar-dotnet/pull/9414#discussion_r1631077590).